### DISCUSSION TITLE
Support in-app purchase via Crypto

### DISCUSSION THREAD

**Discussion by Wjxfi** [ente-io/ente#971] [Enhancements]
A lot of people need this

They are as unaware of this feature as I am. If it were possible in the app, more people would pay that way.

mysterium vpn did it

**Comment by mnvr** [comment]
We already support payments via crypto! 😊

Please see this FAQ entry for more details: https://ente.io/faq/subscription/crypto/

**Comment by Wjxfi** [comment]
> We already support payments via crypto! 😊
> 
> Please see this FAQ entry for more details: https://ente.io/faq/subscription/crypto/

Why can't I do it through the website instead of by email, just like a regular payment?

**Comment by mnvr** [comment]
We do want to do this too! But we also didn't want to get in the way of people who want to pay via crypto right now. 

Integrating payments via the website would require us to write code for each of the providers we have to support. This is definitely the way forward, and but it'll take time, and we don't know when we'd be able to prioritize engineering effort for doing that (we're not a big team, we need to carefully prioritize our time). So we went with the manual approach for now, so that customers are not blocked and have an option for crypto payments.

**Comment by Wjxfi** [comment]
There are sites that accept many cryptocurrencies. one such provider will be enough.

Buying via email is very uncomfortable for me

**Comment by mnvr** [comment]
The payment will be handled via Bitpay or Coinbase already. The only thing manual is that we'll manually generate the invoice for you, instead of automatically generating it from the website.

As in, you'll not pay us directly - we'll generate a Bitpay invoice and send it to your email. When you make the payment (on Bitpay) we'll activate your subscription.

**Comment by Wjxfi** [comment]
Ok. I'll wait until you fully automate (leave the issue open)
